spec/acceptance/docker_full_spec.rb

```diff
@@ -0,0 +1,643 @@
+require 'spec_helper_acceptance'
+
+describe 'Temp OS hacks' do
+  if fact('operatingsystem') =~ /CentOS/
+    if fact('operatingsystemmajrelease') == '7'
+        pp=<<-EOS
+              package {'device-mapper':
+                ensure => latest,
+              }
+        EOS
+      apply_manifest(pp, :catch_failures => true)
+    end
+  elsif default['platform'] =~ /el-7/
+      pp=<<-EOS
+            package {'device-mapper':
+              ensure => latest,
+            }
+
+            package {'docker':
+              install_options => ['--enablerepo=rhel7-extras'],
+              ensure => latest,
+              require => Package['device-mapper']
+            }
+      EOS
+      apply_manifest(pp, :catch_failures => true)
+  end
+end
+
+describe 'docker full acceptance tests section 1 - testing class {\'docker\':}' do
+  context 'test 1.1 - Install and configure Docker and Docker Daemon' do
+    it 'applies the manifest' do
```

Comment:
I'm sorry I don't understand?

The tests are passing clean on RHEL7. CentOS7 (after quickly enabling a normal mirror) and Ubuntu 14.04 consistently